helix-term/src/commands.rs

```diff
@@ -1663,35 +1673,35 @@ fn find_prev_char_impl(
 }
 
 fn find_till_char(cx: &mut Context) {
-    find_char(cx, Direction::Forward, false, false);
+    find_char(cx, Direction::Forward, false);
 }
 
 fn find_next_char(cx: &mut Context) {
-    find_char(cx, Direction::Forward, true, false)
+    find_char(cx, Direction::Forward, true)
```

Comment:
Hmm this sort of breaks the methods: find and extend now are identical and behave implicitly according to the mode. The whole point was to have two distinct commands so the behavior is more predictable when writing macros

Reply:
Yeah, that does stick out like sore thumb. I flip flopped a bit on what I thought made sense here. Options that occurred to me are:

1. the extend flavors to explicitly enter select mode. If you are already in select mode, then this is a no-op.
2. the extend flavors get dropped, and macros are expected to explicitly enter a state, or operate within the existing state.
3. The extend flavors stick around a human readable name, but with no real effect.

IMO the most consistent thing to do is option 2 above, but it will also break custom key maps which is why I left option 3 in place.